### PR TITLE
Fix Sentry SDK initialisation

### PR DESCRIPTION
**Pull request by ushkarev**
and make settings configuration a bit simpler

### CODE REVIEW

**Review by xoen — APPROVED** [review]
As discussed this LGTM, it solves the sentry initialisation issue and it's a bit simpler 👍 

`DEBUG` environment variable may need to be set explicitly on local development environment as it now defaults to `False` (which is the right thing security-wise) - but unless I'm missing something it shouldn't be a problem.

File: mtp_noms_ops/settings/docker.py [hunk 1]
```diff
@@ -4,13 +4,10 @@
 from mtp_common.stack import get_current_pod
 
 from .base import *  # noqa
-from .base import ENVIRONMENT, os
+from .base import DEBUG, ENVIRONMENT, SECRET_KEY, os
```
Comment (xoen) [inline mtp_noms_ops/settings/docker.py:7]: Given we have `from .base import *` above, is this import of `DEBUG` and `SECRET_KEY` for the benefit of your editor or am I missing something here?

**Review by xoen — APPROVED** [review]
LGTM 👍

File: mtp_noms_ops/settings/docker.py (hunk 1 above)
Comment (ushkarev) [reply mtp_noms_ops/settings/docker.py:7]: code inspection tools don't seem to like variables that come from the `*` import 🤷

**Pull request merged**